kubernetes/samples/scripts/common/utility.sh

```diff
@@ -439,13 +452,22 @@ function createFiles {
     sed -i -e "s:%DOMAIN_ROOT_DIR%:${domainPVMountPath}:g" ${deleteJobOutput}
   fi
 
-  if [ "${domainHomeInImage}" == "true" ]; then
+  if [ "${domainHomeSourceType}" == "FromModel" ]; then
```

Comment:
This util script is only used in the domain home on pv or in image samples. Do we support the input file for those use case to have domainHomeSourceType? This line here indicates that the answer is yes, and if this input is specified, we only honor it when the value is 'FromModel', and we ignore the input setting otherwise and honor the old attribute domainHomeInImage. This can cause confusion; the customer may not have other things configured for 'FromModel' using the non-MII samples. We should not allow this in my opinion.

Reply:
Resolved. Dongbo and I added a comment around line 300.  @vanajamukkara FYI